api/kogito-api/src/main/java/org/kie/kogito/rules/DataHandle.java

```diff
@@ -15,6 +15,5 @@
  */
 package org.kie.kogito.rules;
 
-public interface DataHandle {
-    Object getObject();
+public interface DataHandle extends org.drools.ruleunits.api.DataHandle {
```

Comment:
we should probably mark all these interfaces as `@Deprecated` 

```suggestion
/**
 * @deprecated use org.drools.ruleunits.api.DataHandle
 */
 @Deprecated
public interface DataHandle extends org.drools.ruleunits.api.DataHandle {
```

Reply:
I agree, we can also add a note in the release note 👍